data/brands/shop/pet.json

```diff
@@ -9,6 +9,18 @@
     }
   },
   "items": [
+     {
+      "displayName": "L'Isola dei Tesori",
+      "locationSet": {"include": ["it"]},
+      "tags": {
+        "brand": "L'Isola dei Tesori",
+        "brand:wikidata": "Q108578003",
+        "name": "L'Isola dei Tesori",
+        "shop": "pet",
+        "operator": "DMO Pet Care S.r.l.",
+        "pet": "no"
```

Comment:
This is odd because you are adding a pet shop
But if as you said
"Pet shop haven't pet"

Reply:
I've got to put tag sells:pet=no or similar?

Reply:
No, what I means is that If this is a pet shop, why it have ```pet=no``` ?

Reply:
becouse it not sells pet

Reply:
I agree with @LaoshuBaby that `pet=no` is an unusual tag and we should probably remove it.

Also we usually don't add `operator=` tags to the brands, because sometimes shops are operated by a franchisee (what `operator` means is sometimes complicated).

Reply:
ok,  no problem